src/System.CommandLine.Tests/OptionTests.cs

```diff
@@ -370,6 +358,30 @@ public void When_Name_is_set_to_its_current_value_then_it_is_not_removed_from_al
             option.Aliases.Should().Contain("name");
         }
 
+        [Fact]
+        public void When_Name_is_set_to_different_value_then_it_is_not_removed_from_aliases()
```

Comment:
These look like they should ideally be different test cases, potentially one per comment or even one per assertion.

That said, did you find these scenarios were not tested?

Reply:
The general idea is "what happens to aliases when an Option is renamed?"  The earlier assertions are to verify the name change; the later ones are for verifying the state of Aliases.  I hope this helps.

Reply:
It can be hard to figure out whether a specific thing is tested in a large test suite. I _think_ the test you've added is covering scenarios that are already tested but if there's a gap I'd like to close it.

Reply:
Thanks Jon. Let me know if I can help.

Reply:
Thanks, @jburnett.

I feel ok about the existing test coverage here so I'm going to close this.